### ISSUE TITLE
Include an actual changelog in the plugin package

### ISSUE THREAD

**Issue by pixelbrad** [Trumani/stop-spammers#137]
Please include an actual changelog within your plugin - I don't want to be reading through commits to find out what has been changed.. 

Typically the changelog is part of the plugin's `readme.txt` and follows the following format:
```
== Changelog ==

= 1.0.0 - 2019-12-16 =
* Added: some feature
* Updated: plugin functionality for better performance
```

**Comment by bhadaway** [comment]
Hi Brad,

I actually was using one exactly like that before, but I removed it. I'm actually not 100% sure I recall why, but I think it was just becoming a pain to maintain, when I could barely find time to focus on the actual updates themselves.

I added a link to the automated WordPress and GitHub changelogs, hoping that would be enough to cover it, without my having to manually maintain it, but you're of course right, a simple list with a summary of changes is friendlier and easier for users.

Maybe if I'm less precious about mentioning every single change I can just list the most important ones, and it would be less of a pain. Let me think about it.

Thanks
